CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Bumped ONNX version to >= 1.15.0
 * Added support for Leaky ReLU
 * Added support for Pythonic API
+* Remove redundant addends in Equation objects
```

Comment:
So I don't think this should go in the CHANGELOG because it's not an observable change to the user?

Reply:
I see. Sure. I removed it from the CHANGELOG